controllers/operatorpolicy_status.go

```diff
@@ -367,6 +379,7 @@ const (
 	deploymentConditionType  = "DeploymentCompliant"
 	catalogSrcConditionType  = "CatalogSourcesUnhealthy"
 	installPlanConditionType = "InstallPlanCompliant"
+	deprecationType          = "DeprecationWarning"
```

Comment:
The other ConditionTypes we have use "positive polarity" - when the condition is true, that part of the policy is compliant. It would be nice to keep that convention here, so this could be renamed to something like `NoDeprecations`

(The one exception to this is the CatalogSource; we copy the OLM status which uses a negative polarity)